wagtail_streamfield_migration_toolkit/utils.py

```diff
@@ -10,17 +10,21 @@ def __init__(self, *args, instance=None, revision=None, **kwargs):
         super().__init__(*args, **kwargs)
 
     def __str__(self):
+        message = ""
+        message += super().__str__()
```

Comment:
Why not just `= super().__str__()`, and skip the original blank string declaration?
```

Reply:
This doesn't seem to be resolved yet - are we missing a commit?